affiliation_cache.py

```diff
@@ -0,0 +1,47 @@
+import os
```

Comment:
Please add proper docstring explaining what is this script and provide examples how to run it